portfolio/src/main/webapp/index.html

```diff
@@ -56,6 +56,119 @@ <h2 class="subtitle">
     </div>
     <div class="projects-container" id="projects">
       <h1 class="projects-header">Projects</h1>
+
+      <div class="projects-row">
+        <div class="card">
+          <p class="card-title">Samwise</p>
+          <hr />
+          <p class="card-text">
+            Samwise is a homework + schedule planner for students at Cornell
+            University, built with React, Redux, Firebase, and Node.js. Designed
+            and developed by the
+            <a href="https://cornelldti.org">
+              Cornell Design & Tech Initiative.
+            </a>
+          </p>
+          <a
+            href="https://samwise.today"
+            style="background: #0d66ff;"
```

Comment:
Try to put all style attributes in CSS, e.g. add a second class for it. It will make it easier to change stylings in future.

Reply:
Done